Corrected build error in post build

rastrup opened the pull request:
Added Code Analysis and corrected warnings from code analysis
Corrected target framework for deployment

rastrup committed:
Corrected build error in post build
Added Code Analysis and corrected warnings from code analysis
Corrected target framework for deployment

msftclas commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/not_signed)](https://cla.opensource.microsoft.com/Microsoft/Integration?pullRequest=13) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/Microsoft/Integration?pullRequest=13) before we can accept your contribution.<br/><br/>:x: kakkerlakgly [sign now](https://cla.opensource.microsoft.com/Microsoft/Integration?pullRequest=13)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/Microsoft/Integration?pullRequest=13) it.</sub>

rastrup committed:
Corrected Resharper warnings:
Variable hides member
Possible illegal cast
Static member assigned from derived class

rastrup committed:
Removing reduntancies
Typecast
finally blocks
string.Format
Name qualifiers
object.ToString
using directives
comparison with true

rastrup committed:
Code improvements
readonly
HashTable to generic dictionary
for to foreach
move declaration closer to ousage
Naming

rastrup committed:
Sql usage with dispose

rastrup committed:
Null propagation

rastrup committed:
Use object initializers

rastrup committed:
BizTalkAdminOperations cleanup

rastrup committed:
More cleanup
Mostly naming

rastrup committed:
Cleaning up loops

rastrup committed:
Moved to .Net 4.5.2
Updated internal collections
Corrected handling of Exceptions

rastrup committed:
Arrays to LINQ
General collection handling

rastrup committed:
using statements on disposables

rastrup committed:
Minor corrections based on code analysis

rastrup committed:
Extract method for serializing
More precise exceptions
Enum.TryParse
